addOns/spiderAjax/src/main/resources/org/zaproxy/zap/extension/spiderAjax/resources/Messages.properties

```diff
@@ -1,8 +1,32 @@
-# This file defines the default (English) variants of all of the internationalised messages of the plugin
-
-spiderajax.api.action.scan = Runs the spider against the given URL and/or context, optionally, spidering everything in scope. The parameter 'contextName' can be used to constrain the scan to a Context, the option 'in scope' is ignored if a context was also specified. The parameter 'subtreeOnly' allows to restrict the spider under a site's subtree (using the specified 'url').
-spiderajax.api.action.scanAsUser = Runs the spider from the perspective of a User, obtained using the given context name and user name. The parameter 'url' allows to specify the starting point for the spider, otherwise it's used an existing URL from the context (if any). The parameter 'subtreeOnly' allows to restrict the spider under a site's subtree (using the specified 'url').
+# This file defines the default (English) variants of all of the internationalized messages of the plugin
 
+spiderajax.api.view.fullResults = Gets the full spidered content detected by the AjaxSpider. Gives an array on the objects found 
+spiderajax.api.view.numberOfResults = Gets the number of detected resources found on the given url to scan.
+spiderajax.api.view.optionBrowserId = Gets the type of browser being used for the crawl, this has been set at launch time.
+spiderajax.api.view.optionClickDefaultElems = Gets the configured value for "Click Default Elements Only", HTML elements such as "a", "button", "input", all asociated with some action or links on the page.
+spiderajax.api.view.optionClickElemsOnce = Gets the value configured for the AjaxSpider to know if it have to click on the links only once.
+spiderajax.api.view.optionEventWait = Gets the event timer in miliseconds.
```

Comment:
Now that I'm reading through this, the majority of these *view.option* entries may as well align with the content already in the help info: https://github.com/zaproxy/zap-core-help/wiki/HelpAddonsSpiderAjaxOptions